<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="46970" started-at="2019-10-18T15:39:31Z" finished-at="2019-10-18T15:40:17Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="46970" started-at="2019-10-18T15:39:31Z" finished-at="2019-10-18T15:40:17Z">
      <class name="com.atmecs.atmecs_automation.testscripts.TestVerifyFooter">
        <test-method status="PASS" signature="startReport()[pri:0, instance:com.atmecs.atmecs_automation.testscripts.TestVerifyFooter@6d4b1c02]" name="startReport" is-config="true" duration-ms="348" started-at="2019-10-18T15:39:30Z" finished-at="2019-10-18T15:39:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method status="PASS" signature="baseSetup()[pri:0, instance:com.atmecs.atmecs_automation.testscripts.TestVerifyFooter@6d4b1c02]" name="baseSetup" is-config="true" duration-ms="9328" started-at="2019-10-18T15:39:31Z" finished-at="2019-10-18T15:39:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- baseSetup -->
        <test-method status="FAIL" signature="veifyFooterData()[pri:0, instance:com.atmecs.atmecs_automation.testscripts.TestVerifyFooter@6d4b1c02]" name="veifyFooterData" duration-ms="37471" started-at="2019-10-18T15:39:40Z" finished-at="2019-10-18T15:40:17Z">
          <exception class="com.microsoft.sqlserver.jdbc.SQLServerException">
            <message>
              <![CDATA[The column name footer is not valid.]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.microsoft.sqlserver.jdbc.SQLServerException: The column name footer is not valid.
at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(SQLServerException.java:234)
at com.microsoft.sqlserver.jdbc.SQLServerResultSet.findColumn(SQLServerResultSet.java:686)
at com.microsoft.sqlserver.jdbc.SQLServerResultSet.getString(SQLServerResultSet.java:2503)
at com.atmecs.atmecs_automation.utils.DatabaseReader.insertQuery(DatabaseReader.java:30)
at com.atmecs.atmecs_automation.testscripts.TestVerifyFooter.veifyFooterData(TestVerifyFooter.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- com.microsoft.sqlserver.jdbc.SQLServerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veifyFooterData -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.atmecs.atmecs_automation.testscripts.TestVerifyFooter@6d4b1c02]" name="tearDown" is-config="true" duration-ms="136" started-at="2019-10-18T15:40:17Z" finished-at="2019-10-18T15:40:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=veifyFooterData status=FAILURE method=TestVerifyFooter.veifyFooterData()[pri:0, instance:com.atmecs.atmecs_automation.testscripts.TestVerifyFooter@6d4b1c02] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:com.atmecs.atmecs_automation.testscripts.TestVerifyFooter@6d4b1c02]" name="endReport" is-config="true" duration-ms="1052" started-at="2019-10-18T15:40:18Z" finished-at="2019-10-18T15:40:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.atmecs.atmecs_automation.testscripts.TestVerifyFooter -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
